doc/troubleshooting.md

```diff
@@ -609,6 +635,58 @@ Refer to [FAQ: Can names of joints be changed?](faq.md#can-names-of-joints-be-ch
 
 After correcting the configuration, the [changes will need to be propagated to the Yaskawa controller](../README.md#updating-the-configuration).
 
+### Alarm: 8011[56 - 58]
```

Comment:
This is basically the same as for subcodes `[23 - 54]`, right?

Would it be acceptable to merge this section with the previous one?

Reply:
Unfortunately, that would make [55] non-sequential. 
That would definitely cause issues for someone like me.